README.md

```diff
@@ -141,6 +141,25 @@ or human readable tabular formats such as:
 +-----------+----------+-------------------+--------------+---------------------+
 ```
 
+### Installation under MacOS:
+
+First install [Homebrew](https://github.com/Homebrew), a package manager for macOS, if you do not already have it installed. Open terminal and paste:
```

Comment:
rather than giving the install instructions, can just point to homebrew website with install instructions there

```
https://brew.sh/
```